application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -2675,6 +2675,9 @@ public function add_response($sSessionKey, $iSurveyID, $aResponseData)
      * Response to update will be identified either by the response id, or the token if response id is missing.
      * Routine is only applicable for active surveys with alloweditaftercompletion = Y.
      *
+     * If only token and id are provided then token is updated
```

Comment:
I think i prefer to have : 

- if id is provided : this response was updated (token included)
- if only token is provided : only single response with this token was updated.